table/server/common/src/main/java/alluxio/table/common/udb/UdbContext.java

```diff
@@ -98,4 +98,12 @@ public String getUdbDbName() {
   public AlluxioURI getTableLocation(String tableName) {
     return CatalogPathUtils.getTablePathUdb(mDbName, tableName, mType);
```

Comment:
This method cannot return the right AlluxioURI?

Reply:
This method will be deprecated once this PR gets approved, since the directory `/catalog/db/tables` will not be used any more.